src/pfe/file-watcher/server/src/utils/locales/en/translation.json

```diff
@@ -26,7 +26,13 @@
       "missingDeploymentRegistry": "The project will not build due to missing Deployment Registry. Run the Set Deployment Registry command to set a new Deployment Registry to build projects.",
       "invalidDeploymentRegistry": "Codewind was unable to push an image to the Deployment Registry. Please make sure it is a valid Deployment Registry with the appropriate permissions.",
       "wrongDeploymentRegistry": "The project will not build with the new Deployment Registry. Please remove and re-add the project to deploy to the new Deployment Registry.",
-      "unexpectedError": "An unexpected error occurred"
+      "unexpectedError": "An unexpected error occurred",
+      "odoBuildCreateInProgressMsg": "Creating odo component",
+      "odoBuildCreateFailMsg": "Failed to create odo component",
+      "odoBuildPushInProgressMsg": "Building and deploying odo component",
```

Comment:
We don't build and deploy the odo component. Do you mean application?

Reply:
The component (not component type) keyword that I use is from ODO man page: https://github.com/openshift/odo/blob/master/docs/cli-reference.adoc#create. But I can update it to application which makes more sense for our product.